test/unittest/test_tarantool_server.py

```diff
@@ -0,0 +1,43 @@
+import os
```

Comment:
Please add a commit with fix of a problem covered by test to commit where test has been added or at least add a tag for  bug (`Follows up #256`).

Reply:
Fixed.